_targets.R

```diff
@@ -20,44 +20,15 @@ library(maps)
 library(glue)
 library(sf)
```

Comment:
Load packages that are needed to run the pipeline itself at the top of this script (I think just targets is needed right now), and the ones used _in_ the pipeline like 
`
tar_option_set(packages = c("tidyverse", 
                            "raster","sf",
                            "foreign", "ggspatial",
                            'maps','glue')`